docs/ScalarDLBackupCreationRestoration.md

```diff
@@ -0,0 +1,140 @@
+# Scalar DL Backup Creation and Restoration
+
+Scalar DL backup should be created after the Scalar DL server is paused otherwise there is a chance for inconsistency. 
+This guide shows you how to create and restore Scalar DL data backup without any data inconsistency.
+
+## Prerequisites
+
+* Read the Guide on How to [Back up and Restore Databases Integrated with Scalar DB](https://github.com/scalar-labs/scalardb/blob/master/docs/backup-restore.md)
+* Scalar DL must be deployed in the Kubernetes cluster
+* Cosmos DB account must be created with the backup policy `continuous` if you are using Cosmos DB.
+* You must synchronize time on the ledger and auditor servers if you are using the audit service.
+
+## Backup Creation
+
+This section shows how to create a transactionally-consistent backup for Scalar DL.
+
+### Requirements
+
+* You must wait at least 10 sec after pausing to create a backup.
+* You must identify a unique `Point In Time Restoration` for the ledger and auditor if you are using both the ledger and audit services. 
+
+### Find SRV service URL
+
+This section will help you to find the SRV service URL with the help of [SRV records](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#srv-records).
+You can confirm the SRV service URL with the [Debugging DNS Resolution](https://kubernetes.io/docs/tasks/administer-cluster/dns-debugging-resolution/) guide.
+
+Example: 
+```
+_scalardl-admin._tcp.my-release-scalardl-ledger-headless.default.svc.cluster.local
+```
+
+### Create a simple Pod to execute grpcurl
+
+scalaradminutils.yaml
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: scalaradminutils
+  namespace: default
+spec:
+  containers:
+  - name: scalaradminutils
+    image: ghcr.io/scalar-labs/scalar-admin:1.0.0
+    command:
+      - sleep
+      - "3600"
+    imagePullPolicy: IfNotPresent
+  restartPolicy: Always
+```
+
+Use that manifest to create a Pod:
+
+```console
+kubectl apply -f scalaradminutils.yaml
+```
+
+Verify pod status:
+
+```console
+kubectl get pods scalaradminutils
+```
+
+### Pause
+
+Use the following command to pause the Scalar DL service for few seconds
+
+```console
+kubectl exec -i -t scalaradminutils -- ./bin/scalar-admin -c pause -s <SRV_Service_URL>
+```
+Note:-
+* Cosmos DB and DynamoDB will create backup automatically.
+
+### Unpause
+
+Use the following command to unpause the Scalar DL service after at least 10 seconds
+
+```console
+kubectl exec -i -t scalaradminutils -- ./bin/scalar-admin -c unpause -s <SRV_Service_URL>
+```
+
+## Restore
+
+This section shows how to restore transactionally-consistent backup for Scalar DL.
+
+### Requirements
+
+* You must use a middle value of the paused time as restore point.
+* You must restore Scalar Ledger and Auditor tables with the same restore point if you are using the ledger and audit services.
+
+### Cosmos DB
```

Comment:
For each database specific actions, I don't think they're k8s specific so should they be described in the backup doc?

Reply:
Sorry for the inconvenience.
This section has been removed from this guide cd32752